miso-web/src/main/webapp/scripts/hot_pool.js

```diff
@@ -307,45 +313,77 @@ HotTarget.pool = (function() {
-               "Download",
-               [
-                 {
-                   property: "format",
-                   label: "Format",
-                   required: true,
-                   type: "select",
-                   values: Constants.sampleSheetFormats
-                 },
-                 {
-                   property: "instrumentModel",
-                   label: "Instrument Model",
-                   required: true,
-                   type: "select",
-                   getLabel: Utils.array.getAlias,
-                   values: instrumentModels
-                 },
-                 {
-                   property: "pools",
-                   label: "Lanes Configuration",
-                   required: true,
-                   type: "order",
-                   getLabel: Utils.array.getAlias,
-                   values: pools
-                 }
-               ],
-               function(result) {
-                 Utils.ajaxDownloadWithDialog(
-                   "/miso/rest/pools/samplesheet/" + result.format,
+             function showCreateDialog(modelId) {
+               Utils.showDialog(
+                 "Create Samplesheet",
+                 "Download",
+                 [
                    {
-                     instrumentModelId: result.instrumentModel.id,
-                     poolIds: result.pools.map(Utils.array.getId)
+                     property: 'cartridge',
+                     type: 'text',
+                     label: 'Cartridge',
+                     value: '',
+                     required: true
+                   }, {
+                     property: "experimentType",
+                     label: "Type",
+                     required: true,
+                     type: "select",
+                     getLabel: function(type) {
+                       return type.description;
+                     },
+                     values: Constants.illuminaExperimentTypes
+                   },
+                   {
+                     property: "sequencingParameters",
+                     label: "Instrument Model",
+                     required: true,
+                     type: "select",
+                     getLabel: Utils.array.getName,
+                     values: Constants.sequencingParameters.filter(function(param) {
+                         return param.instrumentModel.id == modelId;
+                       })
+                   },
+                   {
+                     property: 'genomeFolder',
+                     type: 'text',
+                     label: 'Genome Folder',
+                     value: Constants.genomeFolder,
+                     required: true
+                   }, {
+                     property: "pools",
+                     label: "Lanes Configuration",
+                     required: true,
+                     type: "order",
+                     getLabel: Utils.array.getAlias,
+                     values: pools
```

Comment:
Just realized that this doesn't allow you to load two lanes of the same pool... it's only one lane per pool. That's probably a separate ticket though.

Reply:
Yeah. It also can't deal with knowing about an S4 flowcell with joined/split lanes. For right now, they are only planning on using this on the MiSeq, so I'm okay with it being limited. I need to see sample sheets for the other platforms with those cases.